src/main/py/peer/peer.py

```diff
@@ -309,47 +313,17 @@ def merge_canvas(self, width: int, height: int, fragment) -> Image.Image:
 
         canvas.save("canvas.png", "PNG")
         return canvas
-    def add_to_art_collection(self, artwork, collection):
-        """
-        Add artwork to collection
-        """
-
-        try:
-            collection.add_to_art_collection(artwork)
-            logging.info("Artwork successfully added to collection.")
-        except ValueError as e:
-            logging.error("Failed to add artwork to collection: %s", e)
-
-    def remove_from_art_collection(self, artwork, collection):
-        """
-        Remove artwork from collection
-        """
-
-        try:
-            collection.remove_from_art_collection(artwork)
-            logging.info("Artwork removed from collection successfully.")
-        except ValueError as e:
-            logging.error("Failed to remove artwork from collection: %s", e)
 
-    def swap_art(self, my_art, their_art, my_art_collection, their_art_collection):
+    async def create_new_ledger(self) -> Ledger:
```

Comment:
Opportunistic

`create_new_ledger_entry` would be more clear to me. This sounds like it's doing `ledger = Ledger()` instead of `ledger.add_owner(...)`